de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/dialogs/ManageMetadataDialog.java

```diff
@@ -28,20 +24,21 @@
 
     private final DiskResourceServiceFacade diskResourceService;
     private final GridView.Presenter.Appearance appearance;
-    private MetadataView.Presenter mdPresenter;
-    private MetadataView mdView;
-
     final DiskResourceUtil diskResourceUtil;
     private boolean writable;
     private DiskResource resource;
     private boolean canHide;
 
+    MetadataView.Presenter mdPresenter;
+
     @Inject
-    ManageMetadataDialog(final DiskResourceServiceFacade diskResourceService,
+    ManageMetadataDialog(MetadataView.Presenter mdPresenter,
+                         final DiskResourceServiceFacade diskResourceService,
```

Comment:
I don't believe the service facade is actually being used in the dialog - can probably remove it.